robottelo/hosts.py

```diff
@@ -561,10 +561,12 @@ def create_custom_repos(self, **kwargs):
             baseurl=http://repourl.domain.com/path
             enabled=1
             gpgcheck=0
-
         """
         for name, url in kwargs.items():
-            content = f'[{name}]\n' f'name={name}\n' f'baseurl={url}\n' 'enabled=1\n' 'gpgcheck=0'
+            content = f'[{name}]\n' f'name={name}\n' f'baseurl={url}\n' 'enabled=1\n' 'gpgcheck=0\n'
+            # Add proxy configuration for IPv6
+            if settings.server.is_ipv6:
+                content += f'proxy={settings.http_proxy.http_proxy_ipv6_url}'
```

Comment:
Just a small suggestion for keeping this same for IPv4 run and just add a newline for proxy for IPv6 run 
```suggestion
            content = f'[{name}]\n' f'name={name}\n' f'baseurl={url}\n' 'enabled=1\n' 'gpgcheck=0'
            # Add proxy configuration for IPv6
            if settings.server.is_ipv6:
                content += f'\nproxy={settings.http_proxy.http_proxy_ipv6_url}'
```